src/core/Popover/Popover.tsx

```diff
@@ -35,14 +35,17 @@ export const Popover = (props: PopoverProps) => {
     matchWidth = true,
     children,
     sourceRef,
+    onClickOutside,
     ...passProps
   } = props;
 
   const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);
 
   const [mountNode, setMountNode] = useState<HTMLElement | null>(null);
 
-  const { styles } = usePopper(sourceRef, popperElement, {
+  const portalRef = useRef<HTMLDivElement>(null);
+
+  const { styles } = usePopper(sourceRef.current, popperElement, {
```

Comment:
This might need further testing / changes due to use of prop ref instead of callback ref and useState as mentioned here: https://popper.js.org/react-popper/v2/.

Reply:
Following issue created and documented the above there.